examples/005_convenience/customize.py

```diff
@@ -6,82 +6,116 @@
 It briefly demonstrates methods that modify the optics of kafe2 output.
 """
 
-from kafe2 import XYContainer, Fit, Plot
+from kafe2 import XYContainer, XYFit, Plot
 
-# The same setup as in 001_line_fit/line_fit.py :
-xy_data = XYContainer(x_data=[1.0, 2.0, 3.0, 4.0],
-                      y_data=[2.3, 4.2, 7.5, 9.4])
+
+# A similar setup to 001_line_fit/line_fit.py
+# This code will execute the fit and plot it without any customizations
+# We do this to demonstrate the difference to the fully customized plot
+def model(x, a, b):
+    return a * x + b
+
+
+xy_data = XYContainer(x_data=[1.0, 2.0, 3.0, 4.0], y_data=[2.3, 4.2, 7.5, 9.4])
 xy_data.add_error(axis='x', err_val=0.1)
 xy_data.add_error(axis='y', err_val=0.4)
-line_fit = Fit(data=xy_data)
-line_fit.do_fit()
+
+fit = XYFit(xy_data=xy_data, model_function=model)
+fit.do_fit()
+fit.report()
+
+plot = Plot(fit_objects=fit)
+plot.plot(residual=True)
+plot.show()
```

Comment:
Avoid multiple calls to `show` in a single example since it will block until the window is closed.